src/sales-channels/asc/managing-orders.md

```diff
@@ -1,71 +1,60 @@
 ---
-title: Managing Orders and Fulfillment
+title: Manage Orders and Fulfillment
 ---
 
 
-Amazon Sales Channel imports your Amazon Seller Central orders into Magento for review on the _Orders_ tab on the Amazon Sales Channel home page.
+Order information received from Amazon Seller Central shows in the _Recent Orders_ section of the [store dashboard]({% link sales-channels/asc/amazon-store-dashboard.md %}).
 
-Orders that are imported:
+Regardless of your [order import settings]({% link sales-channels/asc/order-settings.md %}), Amazon orders that existed in your Amazon Seller Central account prior to your [store integration]({% link sales-channels/asc/store-integration.md %}) will not import into Magento.
 
-- Orders that are in `Unshipped` status in your Amazon Seller Central account at the time of [store integration]({% link sales-channels/asc/store-integration.md %}) during onboarding.
-- All new orders in your Amazon Seller Central account after [store integration]({% link sales-channels/asc/store-integration.md %}).
+## Disabled Order Import
 
-Orders that are not imported:
+After [store integration]({% link sales-channels/asc/store-integration.md %}), [**Import Amazon Orders**]({% link sales-channels/asc/order-settings.md %}#configure-order-settings) is `Disabled` by default. This means that your Amazon order information will show on the store dashboard, but your Amazon orders are not imported and cannot be managed in Magento.
 
-- Orders that existed in your Amazon Seller Central account at the time of [store integration]({% link sales-channels/asc/store-integration.md %}) but were in a status other than `Unshipped`.
+If your [order import settings]({% link sales-channels/asc/order-settings.md %}) are set to `Disabled`, order information received from Amazon appears in the _Recent Orders_ section of the store dashboard. Order information is view only, and managing these orders must be done in Amazon Seller Central.
 
-An Amazon order is added to the to the _Orders_ tab when Amazon Sales Channel receives information that Amazon has changed the order status from `Pending` to `Unshipped` in Amazon Seller Central. Amazon places an order in `Pending` status while the order and the payment method are verified. Remember, information sync depends on your cron settings and the time it takes Amazon to process information.
+![]({% link sales-channels/asc/assets/amazon-recent-orders-not-imported.png %}){: .zoom}
+_Recent Orders - Not Imported_
 
-You can review your Amazon order information on the _Orders_ tab.
+## Enabled Order Import
```

Comment:
Sam as other heading

Reply:
reworded.  denotes text that applies to each setting.